### ISSUE TITLE
Calculate average satisfaction

### ISSUE THREAD

**Issue by lheckmann** [N2-Survey/SurveyFramework#88]
Using the subquestions of the satisfaction question (mostly 'C1', but make a check for the label of the question if it matches), calculate an average satisfaction and add it to the data frame as a separate column.

Computation in 2019: The overall satisfaction was computed by converting the results for each aspect in question C1 to a scale from 1 to 5 ("Very satisfied" = 5, "Very dissatisfied" = 1). No answers or "Does not apply" were set to zero and not taken for the normalisation count. The sum of all satisfaction factor was normalised by the non-zero counts and rounded to the nearest integer values.
Afterwards the integer values were converted back to the original satisfaction scale.

**Comment by lheckmann** [comment]
Yes, this we included in branch 088 because I told Alexandra to write her code in a new file transformation.py (where I would later on move the other transformation codes as well) to make survey.py less crowded.
But it should not affect the main branch right?

**Comment by TheFlike** [comment]
Yep, I just ran the tests on branch 088 and then the tests give an error because n2survey.transformation does not exist,  I now understood that I should have run on branch 96, where the tests work as intended. 
I'm writing with jpietsch now via mail to fix that case. We narrowed it down to either he does not have python V3.9 or he has the corrupted tests from a couple of months back (before simple-simple-comparison) when they were accidently pushed to main. 

Even though Main was rolled back this does not apply to other branches via merge if they already have the corrupted testfiles from main because git thinks during merge that the corrupted files already are the newer ones, so no merge necessary.

Main works fine, just branches that at one time have merged in the corrupted tests from main do not. The solution here is: switch to main, copy out the testscripts of main, switch to your branch and then overwrite the test scripts of your branch with the copied ones.
